accord-core/src/main/java/accord/local/Commands.java

```diff
@@ -286,8 +297,9 @@ public static AcceptOutcome accept(SafeCommandStore safeStore, SafeCommand safeC
         PartialDeps partialDeps = prepareDeps(validated, participants, command, deps);
         participants = prepareParticipants(validated, participants, command);
 
-        safeCommand.accept(safeStore, newSaveStatus, participants, ballot, executeAt, partialTxn, partialDeps, ballot);
+        Command accepted = safeCommand.accept(safeStore, newSaveStatus, participants, ballot, executeAt, partialTxn, partialDeps, ballot);
         safeStore.notifyListeners(safeCommand, command);
+        safeStore.agent().localEvents().onAccepted(safeStore, accepted);
```

Comment:
should this happen before `notifyListeners`?  That can cause other txn to be updated which can fire off first, so the fact that this was `accepted` before that wouldn't be directly visible if you just looked at the events?

Reply:
Sure. I think it makes sense either way, so happy to do it first